examples/ensemble/plot_random_forest_regression_multioutput.py

```diff
@@ -0,0 +1,71 @@
+"""
+============================================================
+Comparing random forests and the multi-output meta estimator
+============================================================
+
+An example to compare multi-output regression with random forest and
+the :ref:`multioutput.MultiOutputRegressor <_multiclass>` meta-estimator.
+
+This example illustrates the use of the
+:ref:`multioutput.MultiOutputRegressor <_multiclass>` meta-estimator
+to perform multi-output regression. A random forest regressor is used,
+which supports multi-output regression natively, so the results can be
+compared.
+
+Using a single underlying feature the model learns both the
+x and y coordinate as output.
+
+"""
+print(__doc__)
+
+# Author: Tim Head <betatim@gmail.com>
+#
+# License: BSD 3 clause
+
+import numpy as np
+import matplotlib.pyplot as plt
+from sklearn.ensemble import RandomForestRegressor
+from sklearn.multioutput import MultiOutputRegressor
+
+
+# Create a random dataset
+rng = np.random.RandomState(1)
+X = np.sort(200 * rng.rand(400, 1) - 100, axis=0)
+y = np.array([np.pi * np.sin(X).ravel(), np.pi * np.cos(X).ravel()]).T
```

Comment:
This should be factored out for train and test so that the user can see they are produced identically.